app/web/src/components/Workspace/WorkspaceAuditLog.vue

```diff
@@ -106,10 +106,10 @@
                 clsx(
                   'h-lg text-sm',
                   rowCollapseState[Number(row.id)]
-                    ? 'bg-action-300'
+                    ? themeClasses('bg-action-200', 'bg-action-900')
```

Comment:
This will also need the hover classes we added, but that is also non-blocking and I will do it in a follow-up PR.